.circleci/config.yml

```diff
@@ -1,551 +1,26 @@
-# Copyright (c) Facebook, Inc. and its affiliates.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
+# Use the latest 2.1 version of CircleCI pipeline process engine.
```

Comment:
Are these changes intentional? These are not included in the PR description.

Reply:
no, not intentional.

I'm not familiar with circleci. I will remove this change